src/npc_class.cpp

```diff
@@ -236,14 +235,32 @@ void npc_class::load( JsonObject &jo, const std::string & )
     optional( jo, was_loaded, "carry_override", carry_override );
     optional( jo, was_loaded, "weapon_override", weapon_override );
 
+
     if( jo.has_array( "traits" ) ) {
-        JsonArray jarr = jo.get_array( "traits" );
+        traits = trait_group::load_trait_group( *jo.get_raw( "traits" ), "collection" );
+    }
+
+    /* Mutation rounds can be specified as either a single distribution:
+     *   "mutation_rounds" : { "constant" : 1 }
+     * or a map of categories to distributions:
+     *   "mutation_rounds" : [
```

Comment:
This example isn't quite right, it looks more like:
```
"mutagen_rounds": [
  { "MUTCAT_ANY": { "constant": 1 } },
  { "MUTCAT_INSECT": { "rng": [1, 3] } }
]
```

Reply:
Good catch, thanks. Fixed in #22494 .